charts/icm-web/values.yaml

```diff
@@ -2,8 +2,6 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 1
```

Comment:
why did you remove the "replicateCount", other defaults are defined at values.yaml too.